pkg/agent/ssh/server.go

```diff
@@ -126,9 +147,27 @@ func (s *Server) handleNonPty(session ssh.Session) {
 
 	fmt.Println("args: ", args)
 
-	cmd := exec.Command("/bin/sh", args...)
+	cmd, err := s.getExecCmd(session.User(), "/bin/sh", args)
+	if err != nil {
+		log.Println("error getting exec cmd: ", err)
```

Comment:
Use `log.Errorf`.

Reply:
@vedranjukic you marked this as resolved but left `Println`

Reply:
I was not aware of the extent of the Println misuse :)